docs/docs/guides/01-basics/02-http/07-file-uploads.md

```diff
@@ -16,7 +16,7 @@ By the end of this guide you will know:
 
 ## Setup
 
-The BodyParser middleware reads the **request body including the uploaded files** and make them available on the `request` object. So, before moving any forward, let's make sure that the middleware is registered under the list of global middleware.
+The BodyParser middleware reads the **request body including the uploaded files** and make them available on the `request` object. So, before moving forward, let's make sure that the middleware is registered under the list of global middlewares.
```

Comment:
The word `middleware` is invariable, so it should stay as `middleware` even it's plural. 😛

Reply:
Thanks for the feedback. Fixed in 12d34bf3fc10f07fda8de29492ebd89dc0389a57 😊